### PR TITLE
Intel.WiFiDrivers version 23.10.0

### LINKED ISSUE #119813: [Package Request]: Windows® 10 and Windows 11* Wi-Fi Drivers for Intel® Wireless Adapters
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.intel.com/content/www/us/en/download/19351/windows-10-and-windows-11-wi-fi-drivers-for-intel-wireless-adapters.html
Publisher: Intel
Package Name: Windows® 10 and Windows 11* Wi-Fi Drivers for Intel® Wireless Adapters
Description: This download record installs the Windows® 10 and Windows 11* WiFi package drivers 22.240.0 for the Intel® Wi-Fi 6E/Wi-Fi 6 and Intel® 9000 series Wireless Adapters.
Package Version: 22.240.0
Installer URL: https://downloadmirror.intel.com/784628/WiFi-22.240.0-Driver64-Win10-Win11.exe

### PR DESCRIPTION
**Pull request by Eddga**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Fixes #119813 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132908)

### CODE REVIEW

**Comment by Eddga** [comment]
Please take a look at my comment in #132906 before approving this PR.

File: manifests/i/Intel/WiFiDrivers/23.10.0/Intel.WiFiDrivers.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: exe
+Scope: machine
+InstallerSwitches:
+  Silent: -q -s
+  SilentWithProgress: -q
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloadmirror.intel.com/794468/WiFi-23.10.0-Driver64-Win10-Win11.exe
+  InstallerSha256: 96A2F4F192B507FB28A9229E2DD14C2EF7109C182933B454534AECC7919D4F8F
```
Comment (stephengillie) [inline manifests/i/Intel/WiFiDrivers/23.10.0/Intel.WiFiDrivers.installer.yaml:17]:
```suggestion
  InstallerSha256: 96A2F4F192B507FB28A9229E2DD14C2EF7109C182933B454534AECC7919D4F8F
- Architecture: x86
  InstallerUrl: https://downloadmirror.intel.com/788770/WiFi-22.160.0-Driver32-Win10.exe
  InstallerSha256: 64F1B31AFD8F842B887C99575494EBE16BB21AFC582688008344A4FA0A3A94DE
```

File: manifests/i/Intel/WiFiDrivers/23.10.0/Intel.WiFiDrivers.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PublisherUrl: https://www.intel.com
+PublisherSupportUrl: https://www.intel.com/content/www/us/en/support.html
+PrivacyUrl: https://intel.com/content/www/us/en/privacy/intel-cookie-notice.html
+PackageName: Intel Wi-Fi Drivers
+PackageUrl: https://www.intel.de/content/www/de/de/download/19351
+License: Proprietary
+LicenseUrl: https://cdrdv2.intel.com/v1/dl/getContent/776972
+Copyright: Copyright (c) Intel Corporation
+CopyrightUrl: https://www.intel.com/content/www/us/en/legal/terms-of-use.html
+ShortDescription: Intel WiFi driver package containing drivers for various WiFi adapters on Windows 10 and 11 64-bit.
```
Comment (stephengillie) [inline manifests/i/Intel/WiFiDrivers/23.10.0/Intel.WiFiDrivers.locale.en-US.yaml:17]:
```suggestion
ShortDescription: Intel WiFi driver package containing drivers for various WiFi adapters on Windows 10 and 11.
```

**Comment by Eddga** [comment]
Thanks for the input and your effort @stephengillie. First I thought 'How did I not think of this earlier?!'.
But Intel doesn't develop the 32-bit version any more and is only providing this last version as is. There will be no update to it anymore. And whenever there is a new version of this manifest due to 64-bit version being updated winget will tell 32-bit users a new version were available while it's still the same version for them. This doesn't necessarily lead to issues but will definitely lead to confusion.
Therefore I think mixing up different versions in one manifest is not a good idea.
I'd suggest reverting your changes and merge the PR unchanged.

**Comment by stephengillie** [comment]
Hi @Eddga,

Glad to help resolve the architecture concern. It sounds similar to Git adding an unavoidable message to their 32-bit version, announcing its depreciation. 

This installer completed normally, but seems to have only installed documentation. 
image

Is this expected? The description suggests that device drivers should be installed also. Does the installer only install driver files if there's a supported adapter attached?

**Comment by Eddga** [comment]
@stephengillie What you said is correct. It exits/installs with success even when there is no suitable hardware installed. So it will probably always install successfully as long as the installer itself doesn't fail somehow.
I also realized that this installer won't create an uninstall entry. As a result winget will never be able to detect the app or detect if updates are available. Thus there is no reason to not also include the x86 version in the package. So I just accepted the changes you suggested earlier.
To make this work better #1417 would have to be implemented. With this it probably could be possible to detect the installed driver version in the future and detect if updates were available.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**